### ISSUE TITLE
I like it but ...

### ISSUE THREAD

**Issue by aurelVZ** [AlanNunes/Compiler#19]
Hello Alan
I like it but ..i must ask you , why in Python ?
I understand that python is somehow easy but is too way slow,and as you can see you get only 
console app in Windows which is GUI OS.
I don't blame you i made few interpreters too but in compiled languages..
to be clear in basic compiler which compile directly into machine code and is easy to lern
like python.
You said you build new programming language ..that is nice!

**Comment by AlanNunes** [comment]
**I appreciate a lot your comment and feedback**

Let me first introduce myself and why I this programming language.

I'm a beginner at Computer Science fields, even though I started programming years ago. It has a few time I decided to study the fields of CS, not only about programming.
I am fascinated about Compilers! One day I wondered myself "how a programming language works? how can i build my own?". Then I started to study and read some pieces of the Dragon Book - "Compilers: Principles, Techniques, and Tools" and search through internet.

I aimed one thing, to build a compiler, no matter it will be fast or whatever. I just wanted to build my own, because it interested me a lot.
That's why I chose Python, because it's so simple, easy and very fast to write programs and my only goal was to build a compiler, just this. I wasn't  to much interested about it to be fast neither about GUI's.

The goal was to build something simple, just for fun and learning.

Thank you for the comment!
